src/sentry/snuba/referrer.py

```diff
@@ -642,6 +642,9 @@ class Referrer(Enum):
         "statistical_detectors.distributions.fetch_transaction_timeseries"
     )
     SUBSCRIPTION_PROCESSOR_COMPARISON_QUERY = "subscription_processor.comparison_query"
+    SUBSCRIPTION_PROCESSOR_COMPARISON_QUERY_PRIMARY = (
+        "subscription_processor.comparison_query.primary"
+    )
```

Comment:
Any tests that should catch issues like this?

Reply:
It depends on the service. I don't really understand how a service could be launched into prod without hitting this check. Every test using this referrer from Sentry should fail.